client/src/components/menubar/authButtons.js

```diff
@@ -19,10 +19,10 @@ const Auth = React.memo((props) => {
           type="button"
           data-testid="auth-button"
           disabled={false}
-          icon={!auth.is_authenticated ? "log-in" : "log-out"}
-          href={!auth.is_authenticated ? auth.login : auth.logout}
+          icon={!userinfo.is_authenticated ? "log-in" : "log-out"}
```

Comment:
```suggestion
          icon={!userinfo["is_authenticated"] ? "log-in" : "log-out"}
```

We have the convention of subscripting non-camel case attributes

Reply:
There's a few other instances of this throughout the diff

Reply:
Was this accidentally resolved? @colinmegill

Reply:
I committed, and am fixing locally